lua/nvim-tree/actions/root/change-dir.lua

```diff
@@ -63,12 +63,12 @@ local function add_profiling_to(f)
 end
 
 M.force_dirchange = add_profiling_to(function(foldername, should_open_view)
-  if should_change_dir() then
+  local isDir = vim.fn.isdirectory(foldername) -- prevent problems on non existing dirs
```

Comment:
No camelcase please: `is_dir`

`is_dir` implies a boolean `local isDir = vim.fn.isdirectory(foldername) == 1`

Reply:
Sorry, I'll change that